tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -107,8 +107,16 @@ static StringRef getTagForParameter(PrintStructureKind context) {
     return "syntaxtype.number";
   case PrintStructureKind::StringLiteral:
     return "syntaxtype.string";
-  }
-  llvm_unreachable("unexpected parameter kind");
+  case PrintStructureKind::DefaultArgumentClause:
+  case PrintStructureKind::DeclGenericParameterClause:
+  case PrintStructureKind::DeclGenericRequirementClause:
+  case PrintStructureKind::EffectsSpecifiers:
+  case PrintStructureKind::DeclResultTypeClause:
+  case PrintStructureKind::FunctionParameterList:
+  case PrintStructureKind::FunctionParameterType:
+    llvm_unreachable("ignored structure kind");
```

Comment:
Maybe we could mention `isIgnoredPrintStructureKind` explicitly either in the unreachable reason or in a comment?